generators/entity-client/index.js

```diff
@@ -127,6 +127,14 @@ module.exports = class extends BaseBlueprintGenerator {
                         `${this.CLIENT_TEST_SRC_DIR}/spec/app/entities/${this.entityFolderName}/${this.entityFileName}.service.spec.ts`
                     );
                 }
+                if (this.isJhipsterVersionLessThan('7.0.0-beta.1') && this.jhipsterConfig.clientFramework === REACT) {
+                    this.removeFile(
+                        `${this.CLIENT_TEST_SRC_DIR}spec/app/entities/${this.entityFolderName}/${this.entityFileName}-management-detail.spec.ts`
+                    );
```

Comment:
```suggestion
```

Why this? There is no `*-management-detail.spec.ts` files generated.
Maybe delete empty `generators\entity-client\templates\react\src\main\webapp\app\entities\entity-management-detail.spec.ts.ejs` to avoid further confusion?

Reply:
yes you're right